src/main/cli/settings.c

```diff
@@ -1006,6 +1009,9 @@ const clivalue_t valueTable[] = {
     { "dterm_notch_hz",             VAR_UINT16 | PROFILE_VALUE, .config.minmaxUnsigned = { 0, FILTER_FREQUENCY_MAX }, PG_PID_PROFILE, offsetof(pidProfile_t, dterm_notch_hz) },
     { "dterm_notch_cutoff",         VAR_UINT16 | PROFILE_VALUE, .config.minmaxUnsigned = { 0, FILTER_FREQUENCY_MAX }, PG_PID_PROFILE, offsetof(pidProfile_t, dterm_notch_cutoff) },
     { "vbat_pid_gain",              VAR_UINT8  | PROFILE_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_PID_PROFILE, offsetof(pidProfile_t, vbatPidCompensation) },
+#if defined(USE_BATTERY_VOLTAGE_SAG_COMPENSATION)
+    { "vbat_sag_compensation",      VAR_UINT8  | PROFILE_VALUE, .config.minmaxUnsigned = { 0, 150 }, PG_PID_PROFILE, offsetof(pidProfile_t, vbat_sag_compensation) },
```

Comment:
Why is the max value for this 150? That's going to allow a 1.5x boost and is going to clip against the max output anyway. Also the documentation doesn't describe the behavior of any value over 100% (and implies this is the max).

Reply:
A compensation value less than 100 reduces the range over which the effect operates.

Conversely, a compensation value over 100 increases the range of operation, and increases the steepness of the response; there will be a greater reduction in motor output with a full battery, and the output will linearly increase until the battery is at minimum. 

With values over 100 there is no clipping any different from usual, it won't clip until 3.5V.

The explanation in the notes above is "Using more than 100% compensation can help whoops that sag really badly, but usually isn't needed."

The main reason why the setting is allowed to 150 is in case it might be useful.  I don't envisage that it would be, but I don't really know for sure, there could be some usage cases where it worked better for people like that.  I'm happy to limit it to 100, or say 120, if, during the RC phase, no-one thinks it's needed.  But at this point it isn't unsafe or anything, and it might be useful, hence it is like it is.